File: c/palindrome.c
#include<stdio.h>
int main()
{
  char a[100];
  int i,k;
  printf("Enter the string:");
  scanf("%[^\n]%*c",a);
  int j = strlen(a);
  for(i=0;i<j;i++)
  {
    if(a[j-1]!=a[i])
      {
        k=1;
        break;
      }
    j--;

  }
  for(i=0;i<j;i++)
  {

  }
  if(k==1)
    {
      printf("%s is palindrome",a);
    }
  else
    {
      printf("%s is not a palindrome",a);
    }
  return 0;
}

File: java/palindrome.java
import java.util.Scanner;

public class palindrome 
{
	public static void main(String arg[]) 
	{
		String str,rev="";
		Scanner s =new Scanner(System.in);
		System.out.println("Enter the String:");
		str =s.nextLine();
		int n=str.length();
		for(int i=n-1;i>=0;i--)
		{
			rev = rev + str.charAt(i);
		}
		if(str.equals(rev))
		{
			System.out.println(str +" is Palindrome");
		}
		else
		{
			System.out.println(str +" is not Palindrome");
		}
	}
	

}
